pep-0008.txt

```diff
@@ -121,13 +121,13 @@ Optional::
       var_one, var_two,
       var_three, var_four)
 
-.. _`multiline if-statements`:
+.. _`multi-line if-statements`:
```

Comment:
I guess, but "multiline" is in pretty common usage, especially when talking about Python's multiline strings.  Chrome's spellchecker doesn't even complain about it.

-0

Reply:
Would you prefer to remove the hyphens in the instances that have them at present? I figure, provided there's not a specific reason for variance, that the form should be consistent within the document.

Reply:
I would prefer it to be "multiline" everywhere.  Thanks.

Reply:
Done.